### DISCUSSION TITLE
whether the guide to using algolia search has been published ?

### DISCUSSION THREAD

**Discussion by iqlal** [thuliteio/doks#834] [Q&A]
whether the guide to using algolia search has been published ?

**Comment by james-d-elliott** [comment]
I don't think so but here's the commit where it was introduced: https://github.com/h-enk/getdoks.org/commit/2c5e0f925c9f6acb4e33df88fd8cdb59d3504b7a

Should be easy enough to extrapolate from there.

**Reply by james-d-elliott** [reply]
I've been working on this (not operational yet): https://github.com/authelia/authelia/pull/4737/files

With the intention that once it's working I'll try make a PR here which swaps FlexSearch for Algolia.

I'd appreciate some guidance from @h-enk on how he configured the Algolia Crawler if that's how he made the index, and maybe the dsn config if he has time. If I can get it all working I'll PR it here and/or make a guide.
